pod.go

```diff
@@ -348,6 +354,20 @@ func (p *Pod) ID() string {
 	return p.id
 }
 
+// Annotations returns any annotation that a user could have stored through the pod.
+func (p *Pod) Annotations(key string) (string, error) {
+	if len(p.config.Annotations) == 0 {
+		return "", fmt.Errorf("Annotations map is empty")
+	}
+
```

Comment:
I don't think this check is necessary. If the map len is empty, the following check (`if exist == false`) will fail.